### ISSUE TITLE
Build error: Could not resolve com.mbeddr:platform:2021.2.+

### ISSUE THREAD

**Issue by bretelerjmw** [discipl/flinteditor-mps#210]
Hi, I tried to build the current repo at master branch so I could read `@doc` strings for the models.

But I ran into a build error that I do not know how to resolve. Can you tell if I'm messing up or if there's a bug?

From the repo root directory:

> `java -version`
```
openjdk version "11.0.2" 2019-01-15
OpenJDK Runtime Environment 18.9 (build 11.0.2+9)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.2+9, mixed mode)
```

>  `./gradlew.bat initProject`

```
> Configure project :
No local.properties file found

FAILURE: Build failed with an exception.

* Where:
Build file 'C:\More Program Files\flinteditor-mps v0.7.1\build.gradle' line: 299

* What went wrong:
A problem occurred evaluating root project 'Flint'.
> Could not resolve all files for configuration ':artifact'.
   > Could not resolve com.mbeddr:platform:2021.2.+.
     Required by:
         project :
      > Failed to list versions for com.mbeddr:platform.
         > Unable to load Maven meta-data from https://projects.itemis.de/nexus/content/repositories/mbeddr/com/mbeddr/platform/maven-metadata.xml.
            > Could not get resource 'https://projects.itemis.de/nexus/content/repositories/mbeddr/com/mbeddr/platform/maven-metadata.xml'.
               > Could not GET 'https://projects.itemis.de/nexus/content/repositories/mbeddr/com/mbeddr/platform/maven-metadata.xml'.
                  > peer not authenticated

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2m 2s
```

**Comment by cblazo** [comment]
It seems that it has to do with this issue: https://github.com/discipl/flinteditor-mps/issues/209

**Comment by TristanAlbers** [comment]
@bretelerjmw 

Got a couple of things to try, could be a temporary server issue, or cache issue:

### Fix 1 (server/cache issue)
1. Clear gradle cache - [explanation for windows](https://stackoverflow.com/questions/23025433/how-to-clear-gradle-cache)
2. run `./gradlew.bat initProject`

### Fix 2 (command issue)
1. Clear gradle cache - [explanation for windows](https://stackoverflow.com/questions/23025433/how-to-clear-gradle-cache)
2. run  `gradlew initProject`

### Fix 3 (proxy issue)
1. Clear gradle cache - [explanation for windows](https://stackoverflow.com/questions/23025433/how-to-clear-gradle-cache)
2. add a system environment variable on windows machine: key = `CI` value = `1` ([how to add system environment variable windows](https://docs.oracle.com/en/database/oracle/machine-learning/oml4r/1.5.1/oread/creating-and-modifying-environment-variables-on-windows.html#GUID-DD6F9982-60D5-48F6-8270-A27EC53807D0))
3. run either of the `initProject` commands

If fix 3 works something funny is going on at the mbeddr server, allowing some connections (both my pcs, both of blackstardlb pcs) bot not all (CI environment, bretelerjmw pc). The proxy could be a fix for all problems, the only caveat is that it does have a rate limit.

**Comment by bretelerjmw** [comment]
Hi, thanks for the suggestions.

I did try to clear cache but the behavior was the same.

However, I noticed that I got slightly different error messages at different attempts. So I actually *used* the caching to eventually download everything I needed.

I did try `CI/1` env variables as well but the behavior was the same.

All in all, I think it was just a very spotty connection to the mps (itemis) server; the "fix" is to wait briefly and rerun the command until you get past the download stage.
